src/test/java/duke/DukeTest.java

```diff
@@ -0,0 +1,17 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Perhaps explicit imports can be used instead of the wildcard imports?

Reply:
Good catch. Thanks and I will fix it !!